configFunctions.f90

```diff
@@ -91,21 +101,24 @@ subroutine findReactionsWithProductOrReactant( r, chs, arrayLen )
     return
   end subroutine findReactionsWithProductOrReactant
 
+  ! ----------------------------------------------------------------- !
```

Comment:
I tend to use 72, although its is not a hard limit. It is true that the comments require a bit of flexibility for clarity. While we are on the aubject ahould every module and subroutine have a header?  (Not to add to your workload, i am happy to do these things as it helps me understand the code better)